### ISSUE TITLE
Loading animated tiles ignores base directory

### ISSUE THREAD

**Issue by eruvanos** [pvcraven/arcade#480]
## Bug Report
Loading animated tiles ignores base directory

### Actual behavior:
```
Traceback (most recent call last):
  File "/Users/maic/workspace/arcade_debug/start.py", line 7, in <module>
    flags.update_animation(150)
  File "/Users/maic/.local/share/virtualenvs/arcade_debug-DkXkyHO6/lib/python3.7/site-packages/arcade/sprite_list.py", line 361, in update_animation
    sprite.update_animation(delta_time)
  File "/Users/maic/.local/share/virtualenvs/arcade_debug-DkXkyHO6/lib/python3.7/site-packages/arcade/sprite.py", line 788, in update_animation
    self.texture = load_texture(source, scale=self.scale)
  File "/Users/maic/.local/share/virtualenvs/arcade_debug-DkXkyHO6/lib/python3.7/site-packages/arcade/draw_commands.py", line 319, in load_texture
    source_image = PIL.Image.open(file_name)
  File "/Users/maic/.local/share/virtualenvs/arcade_debug-DkXkyHO6/lib/python3.7/site-packages/PIL/Image.py", line 2770, in open
    fp = builtins.open(filename, "rb")
FileNotFoundError: [Errno 2] No such file or directory: '../flags/flagGreen2.png'
```

### Expected behavior:
Load animated tiles.

### Steps to reproduce/example code:
Files:

```
.
├── resources
│   ├── flags
│   │   ├── flagGreen1.png
│   │   └── flagGreen2.png
│   └── map
│       └── test_object.tmx
└── start.py
```

* Create a map `test_object.txt`with a TileLayer `flags`.
* Create a Flag animation
* Load the tmx with a basedir `./resources/map`
* Update animation on the processed SpriteList

```
import arcade
map = arcade.tilemap.read_tmx('./resources/map/test_object.tmx')
flags = arcade.tilemap.process_layer(map, 'flags', base_directory='./resources/map')
flags.update_animation(150)
```

**Comment by huooyk** [comment]
Since your test_object.tmx file belongs to a folder called map, not maps, the file cannot be found correctly. If the problem is resolved, please turn off issuses @eruvanos

**Comment by eruvanos** [comment]
@huooyk Thank you for the hint, you are right, I updated the issue to show the actual error.
I opened a new project and oversaw, that it was not showing the expected error.

Makes more sense now?

**Comment by eruvanos** [comment]
Because the map editor itself references everything relative to the map file, I thing to do the same could be a good approach.

That way, a dev would not have to define a base directory.

**Comment by pvcraven** [comment]
Implemented in release 2.2.0
